src/components/Contacts/ContactListTable.jsx

```diff
@@ -37,17 +42,35 @@ const CustomToolbar = () => (
  * @param {object} Props - Props for ContactListTable
  * @param {userListObject[]} Props.contacts - This list of contacts to display
  * @param {Function} Props.deleteContact - Method to delete contact
+ * @param {Function} Props.handleDeleteContact - from Contacts page
+ * @param {Function} Props.addContact - from Contacts page
  * @returns {React.JSX.Element} The ContactListTable Component
  */
-const ContactListTable = ({ contacts, deleteContact }) => {
+const ContactListTable = ({ contacts, deleteContact, handleDeleteContact, addContact }) => {
   const [showMessageModal, setShowMessageModal] = useState(false);
   const [messageToField, setMessageToField] = useState('');
 
+  const [showAddContactModal, setShowAddContactModal] = useState(false);
+  const [contactToEdit, setContactToEdit] = useState({});
+  const [isEditing, setIsEditing] = useState(false);
+
+  const [contacts_, setContacts] = useState(contacts);
```

Comment:
Think we could use `contactsList` as the state being set instead of `contacts_` and `contacts` as the incoming state.